cli/alert_query.go

```diff
@@ -80,7 +80,7 @@ func (a *alertQueryCmd) queryAlerts(ctx context.Context, _ *kingpin.ParseContext
 		// the user wants alertname=<arg> and prepend `alertname=` to
 		// the front.
 		m := a.matcherGroups[0]
-		_, err := labels.ParseMatcher(m)
+		_, err := compat.Matcher(m)
```

Comment:
Just replaced all occurences of `labels.ParseMatcher` with `compat.Matcher`. The switching of the feature happens in `cli/root.go`.